nx-X11/lib/configure.ac

```diff
@@ -0,0 +1,293 @@
+# Initialize Autoconf
+AC_PREREQ([2.60])
+AC_INIT([libNX_X11], [1.6.5],
+        [https://github.com/ArcticaProject/nx-libs/issues], [libNX_X11])
```

Comment:
It's the bugreport URL. May look weird, but is actually correct. https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Initializing-configure.html